### ISSUE TITLE
Add additional scope on facebook

### ISSUE THREAD

**Issue by pcofilada** [adamjmcgrath/react-native-simple-auth#71]
Is it possible to pass additional scope and fields?

**Comment by adamjmcgrath** [comment]
Not at the moment, but it would be pretty trivial to add if you wanted to raise a PR.

You'd need to change https://github.com/adamjmcgrath/react-native-simple-auth/blob/master/lib/providers/facebook.js#L21-L25 to something like:

```js
export const authorize = ({ dance, request }, { appId, callback, scope=SCOPE }) => pipeP(
  dance,
  replace('#', '?'),
  fromQueryString,
)(authorizationUrl(AUTH, appId, callback, scope));
```

Then you could do:

```js
import { facebook } from 'react-native-simple-auth';

facebook({
  appId: ...,
  callback: ...,
  scope: 'some other scope',
}).then(...);
```

**Comment by pcofilada** [comment]
I'm doing it right now. I already added the scope. I'm just having a hard time on adding new fields based on the added scope.

**Comment by pcofilada** [comment]
@adamjmcgrath I added a PR for this.

**Comment by adamjmcgrath** [comment]
Released in [2.1.0](https://travis-ci.org/adamjmcgrath/react-native-simple-auth/builds/220319192#L252)
